test/unit/models/forms/v2/LiabilityDetailsFormSpec.scala

```diff
@@ -115,7 +115,10 @@ class LiabilityDetailsFormSpec extends ModelsBaseSpec {
   )
 
   private val booleanValues = Seq("repaymentClaim")
-  private val emptyParams   = (params -- booleanValues).mapValues(_ => Seq(""))
+
+  private val emptyParams = (params -- booleanValues).map {
```

Comment:
Same with the others as it can just be `(params -- booleanValues).view.mapValues(_ => Seq("")).toMap`. Thanks